osu.Game/Screens/Edit/Screens/Setup/Components/LabelledComponents/LabelledSwitchButton.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Screens.Edit.Screens.Setup.Components.LabelledComponents
+{
+    public class LabelledSwitchButton : CompositeDrawable, IHasCurrentValue<bool>
```

Comment:
I think these components should share a common class, `LabelledComponent` or something.

This and `LabelledTextBox` both share the same corner radius, text, position of the component, colour, layout, etc...

Reply:
All the other components do too, should we first have them merged and then make a PR to organise them accordingly?

Reply:
Should be done from the start.